ngraph/test/files/paddlepaddle/gen_scripts/generate_pow.py

```diff
@@ -0,0 +1,73 @@
+#
+# pow paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+import sys
+
+
+def pdpd_pow(name : str, x, y, data_type):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(name='x', shape=x.shape, dtype = data_type)
+        out = pdpd.fluid.layers.pow(node_x, y, name = 'pow')
+        out = pdpd.cast(out, "float32")
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'x': x},
+            fetch_list=[out])             
+
+        saveModel(name, exe, feedkeys=['x'], fetchlist=[out], inputs=[x], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def pdpd_pow_tensor(name : str, x, y, data_type):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(name='x', shape=x.shape, dtype=data_type)
+        node_y = pdpd.static.data(name='y', shape=y.shape, dtype=data_type)
+        out = pdpd.fluid.layers.pow(node_x, node_y, name='pow')
+        out = pdpd.cast(out, "float32")
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'x': x, 'y': y},
+            fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=['x', 'y'], fetchlist=[out],
+                  inputs=[x, y], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def main():
+    test_cases = [
+        "float32",
+        "int32",
+        "int64"
```

Comment:
1) 'int' tests are failed due to test_case.run_with_tolerance_as_fp, need to test_case.run for 'int' cases
2) For 'int64' I'd suggest to try larger number, like 2^40 and verify that test fails (due to incomplete support of int64 on CPU) - otherwise passed test will look like 'OpenVINO fully supports int64', which is not true

Reply:
Sure, will extend test cases for large int64.
For `int` cases, the new generate script converts the output of model from int to fp32 because the test utils we used from `ngraph/test/util/engine/ie_engine.hpp` only supports fp32 now.
I think the error is caused by an obsolete model named `pow`. I will remove it from `model.csv` because it has been further split to `pow_int32`, `pow_int64`, `pow_float32`

Reply:
I actually can do 'int' inference using 'test_case.run()', so some tests could be executed as int32 i believe

Reply:
Yes, I just figure out the same solution, will check the type of results. If all of them are `int`, then `test_case.run` is used. Thanks

Reply:
resolved in f59d261